lib/react_client/react_interop.dart

```diff
@@ -40,16 +40,6 @@ abstract class ReactDom {
       ReactDOM.unmountComponentAtNode(element);
 }
 
-/// __DEPRECATED.__
-///
-/// Server-side rendering support via APIs in the `react` package will not be maintained beyond the `5.0.0` release.
-@Deprecated('5.0.0')
-@JS('ReactDOMServer')
```

Comment:
Actually, I'm not sure we should be removing this... This actually doesn't relate to the react_server entrypoint (we can still use dart:html with this API) and there's not much we have to do to maintain it.